broadcast-message.php

```diff
@@ -27,14 +27,19 @@
 
 require_once('common.php');
 
-authentication_require_role('global-admin');
+if ($CONF['sendmail_only_global_admin'] == 'YES' ) {
```

Comment:
I can change that, but IIRC the config.inc.php should also be subject of upgrading since all defaults are defined there.